packages/core/src/util.ts

```diff
@@ -95,15 +95,25 @@ export async function getSourceForBacktrace(backtrace: BacktraceFrame[],
   return result
 }
 
-export function runBeforeNotifyHandlers(notice: Notice | null, handlers: BeforeNotifyHandler[]): boolean {
+export function runBeforeNotifyHandlers(notice: Notice | null, handlers: BeforeNotifyHandler[]): { results: ReturnType<BeforeNotifyHandler>[], result: boolean } {
```

Comment:
`ReturnType`! I always forget that this exists! ❤️

Reply:
So handy, and I usually forget as well!